lithops/serverless/backends/k8s/k8s.py

```diff
@@ -345,63 +473,167 @@ def _start_master(self, docker_image_name):
+            self.jobs.append(job_key)
 
-        # Make sure only max_workers are started
-        total_workers = min(max_workers, total_calls // chunksize + (total_calls % chunksize > 0))
+            total_calls = job_payload['total_calls']
+            chunksize = job_payload['chunksize']
+            total_workers = min(max_workers, total_calls // chunksize + (total_calls % chunksize > 0))
 
-        activation_id = f'lithops-{job_key.lower()}'
+            activation_id = f'lithops-{job_key.lower()}'
 
-        job_res = yaml.safe_load(config.JOB_DEFAULT)
-        job_res['metadata']['name'] = activation_id
-        job_res['metadata']['namespace'] = self.namespace
-        job_res['metadata']['labels']['version'] = 'lithops_v' + __version__
-        job_res['metadata']['labels']['user'] = self.user
+            job_res = yaml.safe_load(config.JOB_DEFAULT)
+            job_res['metadata']['name'] = activation_id
+            job_res['metadata']['namespace'] = self.namespace
+            job_res['metadata']['labels']['version'] = 'lithops_v' + __version__
+            job_res['metadata']['labels']['user'] = self.user
 
-        job_res['spec']['activeDeadlineSeconds'] = self.k8s_config['runtime_timeout']
-        job_res['spec']['parallelism'] = total_workers
+            job_res['spec']['activeDeadlineSeconds'] = self.k8s_config['runtime_timeout']
+            job_res['spec']['parallelism'] = total_workers
 
-        container = job_res['spec']['template']['spec']['containers'][0]
-        container['image'] = docker_image_name
-        if not docker_image_name.endswith(':latest'):
-            container['imagePullPolicy'] = 'IfNotPresent'
+            container = job_res['spec']['template']['spec']['containers'][0]
+            container['image'] = docker_image_name
+            if not docker_image_name.endswith(':latest'):
+                container['imagePullPolicy'] = 'IfNotPresent'
 
-        container['env'][0]['value'] = 'run_job'
-        container['env'][1]['value'] = utils.dict_to_b64str(job_payload)
-        container['env'][2]['value'] = master_ip
+            container['env'][0]['value'] = 'run_job'
+            container['env'][1]['value'] = utils.dict_to_b64str(job_payload)
+            container['env'][2]['value'] = master_ip
 
-        container['resources']['requests']['memory'] = f'{runtime_memory}Mi'
-        container['resources']['requests']['cpu'] = str(self.k8s_config['runtime_cpu'])
-        container['resources']['limits']['memory'] = f'{runtime_memory}Mi'
-        container['resources']['limits']['cpu'] = str(self.k8s_config['runtime_cpu'])
+            container['resources']['requests']['memory'] = f'{runtime_memory}Mi'
+            container['resources']['requests']['cpu'] = str(self.k8s_config['worker_processes'])
+            container['resources']['limits']['memory'] = f'{runtime_memory}Mi'
+            container['resources']['limits']['cpu'] = str(self.k8s_config['worker_processes'])
```

Comment:
Please, revert this back to `runtime_cpu`

Reply:
Done